README.md

```diff
@@ -1,16 +1,15 @@
 ## Personal Profiles
 
+Anya Huang- Taiwaness ,2 kids , yoga instructor ,plantbased .
+I have learned [A guide to the Linux terminal for beginners](https://opensource.com/article/21/8/linux-terminal)
```

Comment:
here you could add the heading "What I have learned so far" to structure the page